resources/lang/en/events/relationship_events/breakup_mates.json

```
@@ -0,0 +1,38 @@
+{
+	"had_fight":[
+		"m_c and r_c had a huge fight and broke up.",
+		"At the same spot in camp they announced their commitment to each other as mates, m_c and r_c announce their break up, looking coldly at each other with anger and hurt in their eyes.",
+		"m_c and r_c have been arguing with each other a lot as of late, making everyone uneasy around them. c_n is secretly relieved when they announce that they're no longer mates.",
+		"m_c and r_c had an excellent relationship and were always at each other's side. But to the Clan's surprise, m_c and r_c had a huge fight in the middle of the clearing, exchanging hurtful words before calling off their relationship for good.",
+		"When a Clanmate asked m_c about r_c, m_c hissed angrily at them, saying {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} want to hear anything about that fox-heart again.",
```

```suggestion
		"When a Clanmate asked m_c about r_c, m_c hissed angrily at them, saying {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} want to hear anything about that fox-heart again and they're no longer mates.",
```
It's important to make sure we make the "event" clear. If we don't specify that the mateship ends then the player may misunderstand.